src/ok.rs

```diff
@@ -134,7 +172,12 @@ fn collect_tests() -> Vec<Test> {
                             mesg: None,
                         }
                     }),
-                    sub: vec![],
+                    sub: vec![Test {
+                        name: "SME enabled",
```

Comment:
The name of your test here still doesn't match the names you're giving them in the helper functions. 
For example you have here `SME enabled` and then in the function it's just `SME`. Again, try and keep those consistent.